tests/gtests/test_SW_VegProd.cc

```diff
@@ -174,7 +168,7 @@ namespace {
 
 
 	// Check `get_critical_rank`
-	TEST(VegTest, rank) {
+	TEST_F(AllTest, rank) {

```

Comment:
Please change `TEST_F(VegTest, rank)` to something like `TEST_F(AllTest, VegProdCriticalRanks)`